main/updater/autoUpdater.ts

```diff
@@ -0,0 +1,42 @@
+import { autoUpdater } from 'electron-updater'
+import log from 'electron-log'
+
+import { addCommand, sendError, sendMessage } from '../worker'
+
+const isPrereleaseTrack = process.argv.includes('--prerelease')
+
+autoUpdater.logger = log
+autoUpdater.allowPrerelease = isPrereleaseTrack
+autoUpdater.autoDownload = false
+
+process.on('uncaughtException', sendError)
+autoUpdater.on('error', sendError)
+
+autoUpdater.on('checking-for-update', () => {
+  log.verbose('Performing automatic check for updates', { isPrereleaseTrack })
+})
+
+autoUpdater.on('update-available', res => { //  Ask if they want to download it
+  log.debug('Update available', { res })
+  sendMessage('update', { availableUpdate: { version: res.version, location: 'auto' } })
+})
+
+autoUpdater.on('update-not-available', res => {
+  log.debug('Update not available', { res })
+  sendMessage('update', {})
+})
+
+autoUpdater.on('update-downloaded', res => {
+  log.debug('Update downloaded', { res })
+  sendMessage('update-ready')
+})
+
+addCommand('download', () => {
+  autoUpdater.downloadUpdate()
+})
+
+addCommand('install', () => {
+  autoUpdater.quitAndInstall(true, true)
```

Comment:
You don't need to pass anything here

Reply:
looks like the default for both is `false`

Reply:
Oh this is electron autoupdater not our updater.  Same event names.  Might reduce confusion by calling it electronAutoUpdater or something.